kit/inputs/src/UIKeyTextView.tsx

```diff
@@ -122,10 +121,12 @@ export const UIKeyTextView = React.forwardRef<UIMaterialTextViewRef, UIKeyTextVi
                 onFocus={onFocus}
                 onBlur={onBlur}
                 onChangeText={onChangeText}
-                onKeyPress={onKeyPress}
+                onSubmitEditing={onSubmitEditing}
                 helperText={helperText}
                 success={success}
                 error={error}
+                multiline={false}
```

Comment:
To be honest I'd like this component to be **multiline** (in order to be able to catch the typo if any, otherwise it's very difficult to do)

![Screenshot 2021-11-02 at 13 05 35](https://user-images.githubusercontent.com/3024361/139826633-7a7afa11-ee53-45f7-a373-e2b22016d6ef.png)

But to make it possible, IMHO it's better to show "Send" blue button to the right, instead of the X one. (as we do for UIAddressInput).

In this case no `onSubmitEditing` would be needed. And `onKeyPress` will only be used for Web purposes (to let the user press "Enter")